git-rebase--am.sh

```diff
@@ -4,6 +4,7 @@
 # Copyright (c) 2010 Junio C Hamano.
 #
 
+set -x
```

Comment:
Let's keep this for debugging, so that our manual `git -C 123 reset --hard a5.tag && ./git --exec-path="$PWD" -C 123 -c rebase.usebuiltin=true rebase b1-branch` test will show the command traces by default.

Reply:
Maybe you can separate this out into its own commit, say, with message "DEBUG" so that you remember easily to remove this before you send the patches upstream?

Reply:
I just had a *much better* idea... Why not move that `set -x` into the script snippet in `builtin/rebase.c`? That way, Travis won't complain because it exercises the non-builtin rebase, and you still benefit from seeing the traced commands when testing.